spring-integration-sftp/src/test/java/org/springframework/integration/sftp/inbound/RollbackLocalFilterTests-context.xml

```diff
@@ -31,7 +30,7 @@
 	<bean id="acceptOnceFilter" class="org.springframework.integration.file.filters.AcceptOnceFileListFilter" />
 
 	<int:transaction-synchronization-factory id="syncFactory">
-		<int:after-rollback expression="@acceptOnceFilter.remove(payload)" />
+		<int:after-rollback expression="payload.delete()" />
```

Comment:
Yes, this test fails on OS X even with a 60 second delay - I am not sure why the DELETE event is not received.

I am not sure if the watch service is reliable on any platform - we've introduced a race condition in that the filter removal is now done on a different thread - previously it was done on the poller thread.

In any case, how can this test work now anyway? Given we now populate the remote filter with a persistent accept once filter, the fact that we have deleted the file means the file is removed from the local filter, but it's still in the remote filter.